Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM nvidia/cuda
+
+# Install miniconda
+RUN apt-get update && apt-get install -y wget
+RUN MINICONDA="Miniconda2-latest-Linux-x86_64.sh" && \
+    wget --quiet https://repo.continuum.io/miniconda/$MINICONDA && \
+    bash $MINICONDA -b -p /miniconda && \
+    rm -f $MINICONDA
+ENV PATH /miniconda/bin:$PATH
+
+# Add omnia
+RUN conda config --add channels omnia
+
+# Install deepchem
+RUN conda install --yes deepchem
```

Comment:
The `README.md` suggests you are primarily [recommending users install the "bleeding edge" current git head](https://github.com/deepchem/deepchem#using-a-conda-environment), rather than actual releases. Would it be more useful if the docker image distributed the current git head?